app/scripts/app.js

```diff
@@ -8,7 +8,9 @@
             url: '/'
             , controller: 'HomeCtrl as home'
             , templateUrl: 'templates/home.html'
+        }).state('cookies', {
```

Comment:
What's the purpose of this state? I don't think it offers anything useful. It doesn't even have a URL or a template.